asciidoc/asciidoc.py

```diff
@@ -937,6 +938,8 @@ def next_element():
         # Optimization: Cache answer.
         Lex.prev_cursor = reader.cursor
         Lex.prev_element = result
+        print(f"Lexer: return element: {result}; reader: {reader}")
+        sys.stdout.flush() # needed in github actions for some reason
```

Comment:
[`print`](https://docs.python.org/3/library/functions.html#print) in python 3 has a `flush=True` argument that you can use to get similar functionality as doing `sys.stdout.flush()`.

Though I don't think we should be printing here, rather use `trace` for diagnostic type messages.